@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorMain_v2.sqf

```diff
@@ -0,0 +1,332 @@
+[//-GWT---0k ft---2k ft---4k ft---6k ft---8k ft
+ [ 6804, 1.0126, 1.0221, 1.0168, 1.0172, 1.0237]
+,[ 7711, 1.0058, 1.0042, 1.0028, 1.0007, 1.0108]
+,[ 8165, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000]
+,[ 8618, 0.9948, 0.9962, 0.9921, 0.9938, 0.9855]
+,[ 9525, 0.9964, 0.9947, 0.9835, 0.9832, 0.9792]
+];
+
+private _intTipLossScalarTable = [_tipLossScalarTable, _curGWT_kg] call fza_fnc_linearInterp;
+
+private _tipLossScalarAltTable =
+[
+ [    0, _intTipLossScalarTable select 1]
+,[ 2000, _intTipLossScalarTable select 2]
+,[ 4000, _intTipLossScalarTable select 3]
+,[ 6000, _intTipLossScalarTable select 4]
+,[ 8000, _intTipLossScalarTable select 5]
+];
+
+private _groundEffScalarTable = 
+[//-GWT---0k ft---2k ft---4k ft---6k ft---8k ft
+ [ 6804, 1.2060, 1.1908, 1.1951, 1.2220, 1.2030]
+,[ 7711, 1.1949, 1.1986, 1.2021, 1.2255, 1.2072]
+,[ 8165, 1.2086, 1.1930, 1.1966, 1.2099, 1.2075]
+,[ 8618, 1.2028, 1.1970, 1.2074, 1.2205, 1.2064]
+,[ 9525, 1.2036, 1.1972, 1.1665, 1.1790, 1.1739]
+];
+
+private _intGroundEffScalarTable = [_groundEffScalarTable, _curGWT_kg] call fza_fnc_linearInterp;
+
+private _groundEffScalarAltTable =
+[
+ [    0, _intGroundEffScalarTable select 1]
+,[ 2000, _intGroundEffScalarTable select 2]
+,[ 4000, _intGroundEffScalarTable select 3]
+,[ 6000, _intGroundEffScalarTable select 4]
+,[ 8000, _intGroundEffScalarTable select 5]
+];
+
+private _bladeLiftCoefScalarTable =
+[//-m/s-------SL---2k ft---4k ft---6k ft---8k ft
+ [  0.0,  0.5360, 0.6608, 0.8147, 0.9996, 1.2319]
+,[  5.14, 0.5269, 0.6448, 0.7947, 0.9856, 1.2089]
+,[ 10.29, 0.5343, 0.6526, 0.8081, 0.9887, 1.2180]
+,[ 20.58, 0.5734, 0.7070, 0.8723, 1.0609, 1.3013]
+,[ 27.78, 0.5724, 0.7226, 0.8972, 1.0963, 1.3411]
+,[ 36.01, 0.5732, 0.7051, 0.8752, 1.0689, 1.2966]
+,[ 38.07, 0.5776, 0.7106, 0.8577, 1.0481, 1.2821]
+,[ 46.30, 0.5342, 0.6689, 0.8377, 1.0409, 1.2548]
+,[ 51.44, 0.5046, 0.6370, 0.8646, 0.9846, 1.1893]
+,[ 56.59, 0.4734, 0.5976, 0.7489, 0.9254, 1.1205]
+,[ 61.73, 0.4456, 0.5625, 0.7053, 0.8726, 1.0199]
+,[ 66.88, 0.4393, 0.5581, 0.6956, 0.8565, 1.0009]
+,[ 72.02, 0.4904, 0.6190, 0.7502, 0.9417, 1.1129]
+];
+
+private _intBladeLiftCoefScalarTable = [_bladeLiftCoefScalarTable, _velXY] call fza_fnc_linearInterp;
+
+private _bladeLiftCoefScalarAltTable =
+[
+ [    0, _intBladeLiftCoefScalarTable select 1]
+,[ 2000, _intBladeLiftCoefScalarTable select 2]
+,[ 4000, _intBladeLiftCoefScalarTable select 3]
+,[ 6000, _intBladeLiftCoefScalarTable select 4]
+,[ 8000, _intBladeLiftCoefScalarTable select 5]
+];
+
+private _profileScalar_min = 0.1800;
+private _profileScalar_max = 0.2800;
+
+private _inducedScalarTable =
+[//--Coll--Ind-Min-Ind-Max
+  [ 0.000, 0.7280, 0.2025]
+, [ 1.000, 1.3170, 1.0715]
+];
+
+//Get engine RPM
+(_heli getVariable "fza_sfmplus_engPctNP")
+    params ["_eng1PctNP", "_eng2PctNp"];
+private _inputRPM          = _eng1PctNP max _eng2PctNp;
+//Calculate omega
+private _omega             = (2.0 * PI) * ((_rtrDesignRPM * _inputRPM) / 60);
```

Comment:
`PI` is a script command not a preprocessor macro so should be lower case. Also the parenthesis here are redundant